### PR TITLE
QSPI example for default mode, read-write on flash

### PR DESCRIPTION
**Pull request by deepikabhavnani**
Initial draft read-write on QSPI flash devices

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,179 @@
+#if !DEVICE_QSPI
```
Comment (0xc0170) [inline main.cpp:17]: license header file please on the top of the file

**Review by 0xc0170 — COMMENTED** [review]
CAn you also run this main.cpp file through astyle (I can se eone or two changes that it might do)

File: main.cpp (hunk 1 above)
Comment (deepikabhavnani) [reply main.cpp:17]: done

**Comment by deepikabhavnani** [comment]
@0xc0170 - Please review and merge the example

**Review by 0xc0170 — APPROVED** [review]

**Comment by 0xc0170** [comment]
Sorry about the delay, what else is needed for this example?

**Pull request merged**